protobuf.sh

```diff
@@ -41,11 +42,12 @@ checkHttpRequestCLI() {
 
 downloadFile() {
 
-    FILE_NAME=$1
-    FILE_PATH="${PROTO_PATH}/${FILE_NAME}/v1"
+    FOLDER_NAME=$1
+    FILE_NAME=$2
+    FILE_PATH="${PROTO_PATH}/${FOLDER_NAME}/v1"
```

Comment:
Doesn't this need to be: FILE_PATH="${PROTO_PATH}/${FOLDER_NAME}/v1/{FILE_NAME}" ?

Reply:
We are using FILE_PATH to create directories as needed in that path. Having filename there would create another folder with name {FILE_NAME} which we do not want to happen.